File: trunk/proj/CoreLibrary/src/com/jasonzqshen/familyaccounting/core/exception/BalanceNotZero.java
package com.jasonzqshen.familyaccounting.core.exception;

public class BalanceNotZero extends Exception {

	/**
	 * 
	 */
	private static final long serialVersionUID = 4589373504194478207L;

	public BalanceNotZero() {
		super("Balance is not zero.");
	}

}

File: proj/WinStore_FamilyAccounting/WinStore_FamilyAccountingCore/Exceptions/BalanceNotZero.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WinStore_FamilyAccountingCore.Exceptions
{
    public class BalanceNotZero : Exception
    {
        public BalanceNotZero()
            : base("Balance is not zero.")
        {
        }
    }
}
